contact.rkt

```diff
@@ -43,46 +43,59 @@
               @div[class: "row"]{
                 @h3{Website}
 
-                @div[class: "col-xs-4 pad-contact"]{
-                  @h4{Webmaster} @a[href: "mailto:michael.ballantyne@gmail.com"]{Michael Ballantyne}
-                }
+                @div[class: "row"]{
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{Webmaster} @a[href: "mailto:leif@ccs.neu.edu"]{Leif Andersen}
+                  }
 
-                @div[class: "col-xs-4 pad-contact"]{
-                  @h4{Blog Editor} @a[href: "mailto:moy.cam@husky.neu.edu"]{Cameron Moy}
-                }
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{Blog Editor} @a[href: "mailto:moy.cam@husky.neu.edu"]{Cameron Moy}
+                  }
 
-                @div[class: "col-xs-4 pad-contact"]{
-                  @h4{Publication Page Maintainer} @a[href: "mailto:artem@ccs.neu.edu"]{Artem Pelenitsyn}
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{Publication Page Maintainer} @a[href: "mailto:katiehough19@gmail.com "]{Katie Hough}
+                  }
                 }
 
-                @div[class: "col-xs-6 pad-contact"]{
-                  @h4{People Page Maintainer} @a[href: "mailto:julbinb@gmail.com"]{Julia Belyakova}
-                }
+                @div[class: "row"]{
+                  @div[class: "col-xs-6 pad-contact"]{
+                    @h4{People Page Maintainer} @a[href: "mailto:julbinb@gmail.com"]{Julia Belyakova}
+                  }
 
-                @div[class: "col-xs-6 pad-contact"]{
-                  @h4{Teaching & Software Pages Maintainer} @a[href: "mailto:julbinb@gmail.com"]{Julia Belyakova}
+                  @div[class: "col-xs-6 pad-contact"]{
+                    @h4{Teaching & Software Pages Maintainer} @a[href: "mailto:michael.ballantyne@gmail.com"]{Michael Ballantyne}
+                  }
                 }
               }
 
               @div[class: "row"]{
                 @h3{Social}
 
-                @div[class: "col-xs-3 pad-contact"]{
-                  @h4{Happy Hour} @a[href: "mailto:weiss.a@husky.neu.edu"]{Aaron Weiss}
-                }
-
-                @div[class: "col-xs-3 pad-contact"]{
-                  @h4{Coffee Czar} @a[href: "mailto:types@ccs.neu.edu"]{Ben Greenman}
+                @div[class: "row"]{
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{PRL Social Virtually} @a[href: "mailto:mhyee@ccs.neu.edu"]{Ming-Ho Yee} @br{}
+                    PRL parties, tea time, bingewatching
+                  }
+
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{Twitter @br{} 
+                      @small{@linksym["@neu_prl" "https://twitter.com/neu_prl"]}} 
+                      @a[href: "mailto:mhyee@ccs.neu.edu"]{Ming-Ho Yee}
```

Comment:
@aatxe is now the main Twitter maintainer. (I still have access as backup, but I don't actively maintain it now.)

Reply:
Ah, sure. Somebody, fix this, please!

Reply:
Let me look into it: I think I could improve CSS a little bit on the way.